oac_bylaws.markdown

```diff
@@ -37,9 +37,6 @@ Article IV: Executive Board
 Section I: Composition
 ----------------------
 1. This Board shall be comprised of five officer positions: President-elect, President, Past President, Treasurer and Secretary. This Board will be elected for the first time in the fall of 2011 at the Alumni Weekend or a similar event.
```

Comment:
Strike `This Board will be elected for the first time in the fall of 2011 at the Alumni Weekend or a similar event.`